pkg/packagekit/package_wix.go

```diff
@@ -120,7 +120,8 @@ func PackageWixMSI(ctx context.Context, w io.Writer, po *PackageOptions, include
 
 	if includeService {
 		launcherService := wix.NewService("launcher.exe",
-			wix.WithDelayedStart(),
+			// Ensure that the service does not start until DNS is available, to avoid unrecoverable DNS failures in launcher.
+			wix.WithServiceDependency("Dnscache"),
```

Comment:
How confident are we that this is the correct thing across ALL THE WINDOWS VERSIONS?

Reply:
This service exists for Windows 11, 10, 8, 7, Vista, and XP. Whether it's the best solution for us is still up in the air.

Reply:
That does cover all the versions. 😆

Reply:
It does also appear to work -- launcher + launcher desktop both start up super fast! But I'm hesitant to go for it till I'm more sure that we aren't going to hit a regression w the unrecoverable DNS issue, so that's what I've been working through.